packages/stark-ui/src/modules/table/components/table.component.spec.ts

```diff
@@ -1327,19 +1328,50 @@ describe("TableComponent", () => {
 		});
 	});
 
-	describe("rowClick", () => {
-		const dummyData: object[] = [
-			{ id: 1, description: "dummy 1" },
-			{ id: 2, description: "dummy 2" },
-			{
-				id: 3,
-				description: "dummy 3"
-			}
-		];
+	describe("cellClick", () => {
+		const onClickCallbackSpy = createSpy("onClickCallback");
+	
+		beforeEach(() => {
+			hostComponent.columnProperties = [{ name: "id" }, { name: "description", onClickCallback: onClickCallbackSpy }];
+			hostComponent.dummyData = DUMMY_DATA;
+			hostComponent.rowClickHandler = createSpy("rowClickHandlerSpy", () => undefined); // add empty function so spy can find it
+
+			onClickCallbackSpy.calls.reset();
+			(<Spy>hostComponent.rowClickHandler).calls.reset();
+			
+			hostFixture.detectChanges(); // trigger data binding
+			component.ngAfterViewInit();
+		});
+	
+		it("should trigger 'onClickCallback' property when click on the cell and should not emit on 'rowClicked' when 'onClickCallback' is defined",() => {
+			const descriptionColumnElement = hostFixture.nativeElement.querySelector(
+				"table tbody tr td.mat-column-description"
```

Comment:
Can you add the same check that we have in other tests to ensure the element was found?

```
if (!descriptionColumnElement) {
	fail("No description column element found");
	return;
}
```

well.. in fact the check can be re-written to `expect(descriptionColumn).not.toBeNull();`